radixdlt/src/main/java/com/radixdlt/consensus/LedgerHeader.java

```diff
@@ -66,28 +70,29 @@ private LedgerHeader(
 		@JsonProperty("view") long view,
 		@JsonProperty("accumulator_state") AccumulatorState accumulatorState,
 		@JsonProperty("timestamp") long timestamp,
-		@JsonProperty("isEndOfEpoch") boolean isEndOfEpoch
+		@JsonProperty("next_validators") ImmutableSet<BFTValidator> nextValidators
 	) {
-		this(epoch, View.of(view), accumulatorState, timestamp, isEndOfEpoch);
+		this(epoch, View.of(view), accumulatorState, timestamp, nextValidators);
 	}
 
 	private LedgerHeader(
 		long epoch,
 		View view,
 		AccumulatorState accumulatorState,
 		long timestamp,
-		boolean isEndOfEpoch
+		ImmutableSet<BFTValidator> nextValidators
 	) {
 		this.epoch = epoch;
 		this.view = view;
 		this.accumulatorState = accumulatorState;
-		this.isEndOfEpoch = isEndOfEpoch;
+		this.nextValidators = nextValidators;
 		this.timestamp = timestamp;
 	}
 
-	public static LedgerHeader genesis(Hash accumulator) {
+	public static LedgerHeader genesis(Hash accumulator, BFTValidatorSet nextValidators) {
```

Comment:
See below comment about LedgerHeader.create() method.

Reply:
why not use Optional here, rather than nulls?